views/jira-configuration.hbs

```diff
@@ -1,143 +1,193 @@
-      </header>
+    {{#if hasConnections}}
+      <button
+          class="jiraConfiguration__header__title__cta ak-button ak-button__appearance-primary aui-button aui-dropdown2-trigger"
+          aria-controls="connectBtnDropdown"
+      >Connect GitHub organization
+      </button>
+      <aui-dropdown-menu id="connectBtnDropdown">
+        <aui-section>
+          <a
+            class="jiraConfiguration__connectBtnDropdownOption add-organization-link"
+            target="_blank"
+          >
+            <span>Connect GitHub Cloud</span>
+            <i class="aui-icon aui-iconfont-question-filled"></i>
+            <div class="jiraConfiguration__connectGHCloudTooltip">Select this option if the URL of your GitHub organization starts with "http://github.com".
+              This includes “Free”, “Team” and “GitHub Cloud Enterprise” accounts.</div>
+          </a>
+          <a
+            class="jiraConfiguration__connectBtnDropdownOption select-github-version-link"
+            target="_blank"
+          >
+            <span>Connect GitHub Server</span>
+            <i class="aui-icon aui-iconfont-question-filled"></i>
+            <div class="jiraConfiguration__connectGHETooltip">Select this option if you use the on-premise version of GitHub installed into your own infrastructure.
+              The URL for your GitHub organization starts with your own unique domain.</div>
+          </a>
+        </aui-section>
+      </aui-dropdown-menu>
 
-      <section class="jiraConfiguration_content">
-        {{#if hasConnections}}
-          <p>
-            After you connect your GitHub account or organization to Jira and
-            include issue keys in branch names, commit messages, and pull
-            request titles, you will be able to:</p>
-          <ul class="jiraConfiguration__content__list">
-            <li>Create pull requests from your Jira project.</li>
-            <li>View development information within your linked Jira issue.</li>
-            <li>View deployment pipelines over a timeline in your insights.</li>
-          </ul>
-          <p>Even if the backfill status is still in progress, you can start linking
-            issue keys in your development work immediately.</p>
+    {{/if}}
+  </header>
 
-         {{!-- Backfill Status Info Modal --}}
-          <div class='jiraConfiguration__syncRetryModal'  id="sync-status-modal">
-            <div class='jiraConfiguration__syncRetryModalContent'>
-              <div class="jiraConfiguration__syncRetryModal__headerContainer">
-                <h4 class="jiraConfiguration__syncRetryModal__header">Backfill status</h4>
-                <span id="status-close" class="jiraConfiguration__syncRetryModal__closeBtn aui-icon aui-iconfont-close-dialog"></span>
-              </div>
+  <section class="jiraConfiguration_content">
+    {{#if hasConnections}}
+      <p>
+        After you connect your GitHub account or organization to Jira and include issue keys in branch names, commit
```

Comment:
Hi this file has big differences from `git` 's point of view, do you mind re-structure the whitespace and align to main branch as much as possible? 

--------

I think generally, even there's formatting issue in the origin code, if we could not touch it, the PR (git diff) can more focus on the change.

Reply:
2 Partials created to shorten the changes on this file